scripts

```diff
@@ -0,0 +1 @@
+tfmod-scaffold/scripts
```

Comment:
My bad, I should ignore this soft link in .gitignore file. Would you please remove this file from this pr? Thanks!

Reply:
Done, sorry didn't notice that!